src/main/menu/templates/help.js

```diff
@@ -32,32 +32,27 @@ export default function () {
     label: '&Help',
     role: 'help',
     submenu: [{
-      label: 'Learn More',
+      label: 'Quick Start',
       click () {
-        shell.openExternal('https://marktext.app')
-      }
-    }, {
-      label: 'Source Code on GitHub',
-      click () {
-        shell.openExternal('https://github.com/marktext/marktext')
+        shell.openExternal('https://github.com/marktext/marktext/blob/develop/docs/README.md')
```

Comment:
We should use `master` before stable release for stable version and add latest and stable documentation to wiki.

Reply:
So I need to change `develop` to `master`？

Reply:
You can or we change this after the release candidates are finished because not documentation exist on master.

Reply:
>because not documentation exist on master.

I think it's not a problem(because we did not release it, so no one will view the help new menu), I'll change it to `master`.